Devices/igtlioVideoDevice.cxx

```diff
@@ -174,7 +191,7 @@ igtl::MessageBase::Pointer VideoDevice::GetIGTLMessage()
 //---------------------------------------------------------------------------
 igtl::MessageBase::Pointer VideoDevice::GetIGTLMessage(MESSAGE_PREFIX prefix)
 {
- if (prefix==MESSAGE_PREFIX_START)
+/*if (prefix==MESSAGE_PREFIX_RTS)
```

Comment:
If this is not used, remove the code. Old code exists in the commit history.

Reply:
will do